compiler/testData/codegen/box/diagnostics/functions/tailRecursion/unitBlocks.kt

```diff
@@ -7,14 +7,14 @@
 // IGNORE_BACKEND: JS
```

Comment:
BTW do we know what's wrong with JS_IR here?

Reply:
The test framework isn't stripping out the diagnostics.

>Unresolved reference: NON_TAIL_RECURSIVE_CALL

Reply:
As I know @vana06 migrating some of JS tests to new infrastructure and hopefully it would be fixed automatically after migration

Reply:
Got it, thank!